src/drivers/intel/cavs/ipc.c

```diff
@@ -36,6 +36,9 @@
 
 #endif
 
+/* 8k cycles per systick */
+#define CAVS_IPC_TASK_CYCLES_BUDGET 8000
```

Comment:
systick should be set in kconfig as it is in Zephyr, btw why are we chossing 8k today ?

Reply:
It's 8MCPS when systick is set for 1ms, it was just empirically proven to work ok for both LPRO/HPRO configs over the years in other FWs, IDK if there were any advanced calculations irt it. But let's assume we have LPRO (120MCPS to use), then it will be 8/120 processing power reserved for IPC. Most of IPCs get handled in 1/2 ms with budgeting what is a lot more than our timeout value (300ms if I remember correctly).